<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPhoto Camera Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Camera Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>GPhoto Version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_version_local">
          <property name="text">
           <string>VERSION</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_model">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Driver Quality</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_quality">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Number of Images left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_images_left">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Space Total</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_space_total">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Space Left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_space_left">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="styleSheet">
      <string notr="true">QLabel:disabled {
	color: rgb(211, 211, 211);
}</string>
     </property>
     <property name="title">
      <string>Camera Abilities</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="lb_capture_image">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Capture Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_capture_image">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Try</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="lb_capture_audio">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Capture Audio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_capture_audio">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Try</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="lb_capture_preview">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Capture Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_capture_preview">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Try</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="lb_capture_movie">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Capture Movie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_capture_movie">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Try</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="lb_driver_configuration">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Driver Config</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_driver_configuration">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Try</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_read_settings">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Show read settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Logging</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="log">
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="centerOnScroll">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QPushButton" name="pb_rescan">
       <property name="text">
        <string>Rescan for Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_close">
       <property name="text">
        <string>Quit</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>InfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>709</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>703</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
